test/erlfmt_SUITE_data/comments.erl.formatted

```diff
@@ -3,6 +3,7 @@
 -export([has_fanciness/1, bar/1, baz/1]).
 
 %% Constants
+
```

Comment:
This behaviour seems counter-intuitive to me i.e.:

%% commmentcommentcomment
long line of thing... % comment

becoming:

%% commmentcommentcomment

% comment
long line of thing...

Reply:
Yeah, I wasn't 100% sure about this. What would be the desired behaviour? Something like this?

```
%% commmentcommentcomment
% comment
long line of thing...
```

Reply:
Yeah, that would be my intuition